tests/pytests/functional/transport/zeromq/test_pub_server_channel.py

```diff
@@ -204,13 +204,13 @@ def __exit__(self, *args):
 @pytest.mark.slow_test
 def test_publish_to_pubserv_ipc(salt_master, salt_minion):
     """
-    Test sending 10K messags to ZeroMQPubServerChannel using IPC transport
+    Test sending 10K messages to ZeroMQPubServerChannel using IPC transport
 
     ZMQ's ipc transport not supported on Windows
     """
     opts = dict(salt_master.config.copy(), ipc_mode="ipc", pub_hwm=0)
     with PubServerChannelProcess(opts, salt_minion.config.copy()) as server_channel:
-        send_num = 10000
+        send_num = 4
```

Comment:
I should undo this change. This number was reduced to easier testing and comparison of rabbitmq and zeromq.